scripts/saml-configuration

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import io
+import logging
+import os
+import pwd
+import sys
+from typing import Any, Dict, Iterator, List, Tuple
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(BASE_DIR)
+# Enter the Zulip virtual environment.
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+# Settings for django.conf
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
```

Comment:
What's the thinking behind making this a scripts/ tool rather than a management command?  I think `manage.py configure_saml` would be a more natural mechanism, and it'd let us avoid the extra copy of all the Django setup boilerplate.